models/articles.models.js

```diff
@@ -1,15 +1,41 @@
 const db = require("../db/connection");
-const { checkUserExists } = require("../utils/utils.js");
+const { checkItemExists } = require("../utils/utils.js");
 
-exports.fetchArticles = () => {
-  const queryString = `
+exports.fetchArticles = (sort_by, order, topic) => {
+  const validColumns = [
+    "article_id",
+    "title",
+    "topic",
+    "author",
+    "body",
+    "created_at",
+    "votes",
+  ];
+  if (!validColumns.includes(sort_by)) sort_by = "created_at";
```

Comment:
Long comment for a small detail:

We could assign default values up on line 4 instead of testing and assigning them here. Simply by using `order = "desc"` as the parameter, this gives the parameter `order` the default value of `"desc"` if the argument passed is undefined.  You could do similarly for the `sort_by` parameter.  Or, indeed, any other future parameter you wish to have a default value in your future coding life 😄 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters